ecc/blocks/event-info-component/controller.js

```diff
@@ -190,13 +190,41 @@ export function onSubmit(component, props) {
   setPropsPayload(props, eventInfo);
 }
 
+const disableMczEventFields = (component, disabled = true) => {
```

Comment:
Calling a function **disableSomething** and have an **optional boolean parameter** to dictate whether it's disabling or not is not the best practice...You can call this function toggleDisableMczEventFields and make the boolean param required, or better yet, make 2 functions.